src/Stripe.net/Constants/Events.cs

```diff
@@ -274,7 +274,7 @@ public static class Events
         public const string CustomerSubscriptionTrialWillEnd = "customer.subscription.trial_will_end";
 
         /// <summary>
-        /// Occurs three days before the trial period of a subscription is scheduled to end.
+        /// Occurs whenever a customer updates their subscription.
```

Comment:
thanks for the change! Could you tweak the description to match the docstring from https://stripe.com/docs/api/events/types#event_types-customer.subscription.updated